src/main/java/com/fundation/webservice/controller/Controller.java

```diff
@@ -335,4 +335,146 @@ public KeyFrameResponse convertKeyFrame(@RequestParam("asset") MultipartFile ass
+        JSONObject inputJson = new JSONObject(input);
+        JSONObject configJson = new JSONObject(config);
+        JSONObject outputJson = new JSONObject(output);
+        String fileName = uploadService.storeFile(asset);
+        String fileDownloadUri = ServletUriComponentsBuilder.fromCurrentContextPath().path("/download/").path(outputJson.getString("name")
+                + ".zip").toUriString();
+        String inputChecksumString = "";
+        InputStream inputProperties;
+        try {
+            inputProperties = new FileInputStream("application.properties");
+            properties.load(inputProperties);
+        } catch (IOException io) {
+            io.printStackTrace();
+        }
+
+        try {
+            inputChecksumString = checksum.getChecksum(properties.getProperty("file.uploadDir") + asset.getOriginalFilename(),
+                    "MD5");
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+        if (inputJson.getString("checksum").equals(inputChecksumString)) {
+            new File(properties.getProperty("file.downloadDir") + outputJson.getString("name") + "/").mkdirs();
+            CriteriaImage criteria = new CriteriaImage();
+            criteria.setSrcPath(properties.getProperty("file.uploadDir") + fileName);
+            criteria.setDestPath(properties.getProperty("file.downloadDir") + outputJson.getString("name") + "/" +
+                    outputJson.getString("name") + outputJson.getString("ext"));
+            ConvertImage converter = new ConvertImage();
+            converter.convert(criteria);
+            String outputChecksumString = "";
+            try {
+                outputChecksumString = checksum.getChecksum(properties.getProperty("file.downloadDir") +
+                        outputJson.getString("name") + "/" + outputJson.getString("name") +
+                        outputJson.getString("ext"), "MD5");
+            } catch (Exception e) {
+                e.printStackTrace();
+            }
+            if (configJson.getString("metadata").equals("json")) {
+                //Creation JSON
+                File convertedFile = new File(properties.getProperty("file.downloadDir") +
+                        outputJson.getString("name") + "/" + outputJson.getString("name") +
+                        outputJson.getString("ext"));
+                Metadata metaDataFile = new Metadata();
+                metaDataFile.writeJsonFile(convertedFile);
+            } else {
+                //Creation XMP
+                File convertedFile = new File(properties.getProperty("file.downloadDir") +
+                        outputJson.getString("name") + "/" + outputJson.getString("name") +
+                        outputJson.getString("ext"));
+                Metadata metaDataFile = new Metadata();
+                metaDataFile.writeXmpFile(convertedFile);
+            }
+
```

Comment:
Remove this line.

Reply:
Adopted as this group standard, this line is inserted in order to make readibility easier. At this point we want to stress the difference between the operations carried before the line and after the line.
Conclusion: This blank line will be preserved.